utils/directories.cc

```diff
@@ -190,6 +190,10 @@ static seastar::sstring to_sstring(const fs::path& p) {
     return seastar::sstring{p.c_str()};
 }
 
+seastar::sstring directories::get_work_dir() const {
```

Comment:
This looks like runaway part from patch [Allow utils::directories to provide paths to dirs](https://github.com/scylladb/scylladb/pull/16787/commits/0b88fe8d6c7725899736b8900915ada7e1d7711a) of this PR. Please, either introduce _each_ `get_foo_dir()` getter one-by-one in the same patches that start using it, or introduce _all_ `get_foo_dir()` getters in one preparation patch

Reply:
Sure, fixed. The content of this commit has been moved to the change, that adds the new interface. Thanks :+1: